index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <header id="header">
+      <h1>phones</h1>
+      <img src="./header-img.jpg" alt="header-img" id="header-img" />
+      <nav id="nav-bar">
+        <ul>
+          <li><a class="nav-link" href="#features">article1</a></li>
+          <li><a class="nav-link" href="#how-it-works">article2</a></li>
+          <li><a class="nav-link" href="#pricing">article3</a></li>
```

Comment:
you could just have the class on the <ul>

Reply:
Can you please complete this sentence? Also, it would better to address your points politely and clearly. This way the changes you are suggesting are more clear.